encodedcc.py

```diff
@@ -662,8 +662,18 @@ def patch_set(args, connection):
                     if k[1] == "int" or k[1] == "integer":
                         patch_data[k[0]] = int(temp_data[key])
                     elif k[1] == "array" or k[1] == "list":
+                        if type(temp_data[key]) == str:
+                            # So JSON loads if present.
+                            temp_data[key] = temp_data[key].replace("'",'"')
```

Comment:
Exactly. If you store the JSON in your patch list with single quotes it will break when it loads (since it loads as a string from DictReader). This feels hacky but not sure of proper way to deal with this.
<img width="702" alt="screen shot 2017-08-01 at 2 58 37 pm" src="https://user-images.githubusercontent.com/16053922/28849041-139503fc-76ca-11e7-8e81-dd0326d05afe.png">